modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/BucketAccessor.java

```diff
@@ -45,6 +46,9 @@
     @Query("select * from bucket where experiment_id = ?")
```

Comment:
yes, we could and should bit in this feature I am trying to keep scope limited to Batch assignment only...